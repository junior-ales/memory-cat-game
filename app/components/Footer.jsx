import React from 'react';
import { compose } from 'ramda';

const Footer = text => <div>{text}</div>;

export default compose(Footer, props => props.year);
