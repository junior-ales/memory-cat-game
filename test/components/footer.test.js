/* eslint-disable */
import renderer from 'react-test-renderer';
import React from 'react';
import Footer from '../../app/components/Footer';

const props = {
  year: '2016'
};

test('should render footer', () => {
  const renderedComponent = renderer.create(
    <Footer {...props} />
  );

  expect(renderedComponent.toJSON()).toMatchSnapshot();
});
