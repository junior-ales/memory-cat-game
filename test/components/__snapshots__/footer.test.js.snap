exports[`test should render footer 1`] = `
<div>
  2016
</div>
`;
